lib/tasks/jsminify.js

```diff
@@ -5,6 +5,18 @@
  */
 var yuglify = require('yuglify');
 
+var yuglifyConfig = {
+    mangle: true,
+    output: {
+      semicolons: true,
```

Comment:
I had to copy the config to override `semicolons` to `true`.  Old yuglify (<2.0.0) was using wrong name `semicolon`.  yuglify@2.0.0 fixed it, so that `semicolon: false` became effective.

In yuglify, we would like to preserve the old behavior, where a semicolon is present at the end of minified file.  This is to prevent potential issue when assets are being combo-ed.